src/js/tooltip.js

```diff
@@ -28,7 +28,7 @@ class Tooltip {
 		this.visible = false;
 
 		this.delegates = {
-			target: new Delegate(this.target.targetEl),
+			target: new Delegate(document.getElementById(this.opts.target)),
```

Comment:
Why was this changed? We already have a reference to the target element in `this.targetEl`.